app/src/main/res/values/strings_koala.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="Activity_Load_More">Activity Load More</string>
```

Comment:
Let's have a lil chat on these fellas on Tuesday. I think a super simple approach would be to just have a class with a bunch of static strings. We won't have a need to localize these events, and using static strings means we avoid the need to pass around a context.